---
abstract: Linking distal enhancers to genes and modeling their impact on target gene expression are longstanding unresolved problems in regulatory genomics and critical for interpreting non-coding genetic variation. Here we present a new deep learning approach called GraphReg that exploits 3D interactions from chromosome conformation capture assays in order to predict gene expression from 1D epigenomic data or genomic DNA sequence. By using graph attention networks to exploit the connectivity of distal elements and promoters, GraphReg more faithfully models gene regulation and more accurately predicts gene expression levels than dilated convolutional neural networks (CNNs), the current state-of-the-art deep learning approach for this task. Feature attribution used with GraphReg accurately identifies functional enhancers of genes, as validated by CRISPRi-FlowFISH and TAP-seq assays, outperforming both CNNs and the recently proposed Activity-by-Contact model. GraphReg therefore represents an important advance in modeling the regulatory impact of epigenomic and sequence elements.
authors:
- admin
- Merve Sahin
- Christina S. Leslie
date: "2022-04-08T00:00:00Z"
doi: "10.1101/gr.275870.121"
featured: false
publication: "Genome Research"
publication_short: "Genome Res"
publication_types:
- "2"
publishDate: "2022-04-08T00:00:00Z"
title: Chromatin interaction aware gene regulatory modeling with graph attention networks
url_pdf: https://www.biorxiv.org/content/10.1101/2021.03.31.437978v1.full.pdf
---
