---
abstract: Identifying transcriptional enhancers and their target genes is essential for understanding gene regulation and the impact of human genetic variation on disease. Here we create and evaluate a resource of >13 million enhancer-gene regulatory interactions across 352 cell types and tissues, by integrating predictive models, measurements of chromatin state and 3D contacts, and large-scale genetic perturbations generated by the ENCODE Consortium. We first create a systematic benchmarking pipeline to compare predictive models, assembling a dataset of 10,411 element-gene pairs measured in CRISPR perturbation experiments, >30,000 fine-mapped eQTLs, and 569 fine-mapped GWAS variants linked to a likely causal gene. Using this framework, we develop a new predictive model, ENCODE-rE2G, that achieves state-of-the-art performance across multiple prediction tasks, demonstrating a strategy involving iterative perturbations and supervised machine learning to build increasingly accurate predictive models of enhancer regulation.
authors:
- admin
- Multiple ENCODE Consortium Authors
date: "2023-11-13T00:00:00Z"
doi: "10.1101/2023.11.09.563812"
featured: false
publication: "bioRxiv"
publication_short: "bioRxiv"
publication_types:
- "3"
publishDate: "2023-11-13T00:00:00Z"
title: "An encyclopedia of enhancer-gene regulatory interactions in the human genome"
url_pdf: https://www.biorxiv.org/content/10.1101/2023.11.09.563812v1.full.pdf
---